src/cpu/gemm_inner_product_utils.hpp

```diff
@@ -91,15 +91,18 @@ struct pp_kernel_t {
     bool runtime_mb() const { return MB_ == (size_t)DNNL_RUNTIME_DIM_VAL; }
 };
 
-static const bcast_set_t gemm_default_strategies {
-        broadcasting_strategy_t::scalar, broadcasting_strategy_t::per_oc,
-        broadcasting_strategy_t::per_oc_spatial,
-        broadcasting_strategy_t::no_broadcast};
+static const bcast_set_t &gemm_default_strategies() {
```

Comment:
Clang issues a warning, I think using just `inline` would be sufficient.
```
'static' function 'gemm_default_strategies' declared in header file should be declared 'static inline' [-Werror,-Wunneeded-internal-declaration]
static const bcast_set_t &gemm_default_strategies() {
```

Reply:
I did that. Didn't I push the update?